Formula/avimetaedit.rb

```diff
@@ -0,0 +1,25 @@
+class Avimetaedit < Formula
+  desc "A tool that supports embedding, validating, and exporting of metadata in AVI files"
+  homepage "https://mediaarea.net/AVIMetaEdit"
+  url "https://mediaarea.net/download/binary/avimetaedit/1.0.1/AVIMetaEdit_CLI_1.0.1_GNU_FromSource.tar.bz2"
+  version "1.0.1"
+  sha256 "3aa4ca6aed51c71684f3fd1bd3b4defc2f9e50a8bd0bdf10e55e86c8ba24aafd"
+
+  def install
+    cd "Project/GNU/CLI" do
+      system "./configure",  "--disable-debug", "--prefix=#{prefix}"
+      system "make", "install"
+    end
+  end
+
+  test do
+    avi = "UklGRuYAAABBVkkgTElTVMAAAABoZHJsYXZpaDgAAABAnAAAlgAAAAAAAAAQCQAAAQAAAAAAAAABAAAAAAAQAA" \
+          "IAAAACAAAAAAAAAAAAAAAAAAAAAAAAAExJU1R0AAAAc3RybHN0cmg4AAAAdmlkc0k0MjAAAAAAAAAAAAAAAAAB" \
+          "AAAAGQAAAAAAAAABAAAABgAAAP////8AAAAAAAAAAAIAAgBzdHJmKAAAACgAAAACAAAAAgAAAAEADABJNDIwBg" \
+          "AAAAAAAAAAAAAAAAAAAAAAAABMSVNUEgAAAG1vdmkwMGRjBgAAABAQEBCAgA==".unpack("m")[0]
+
+    File.open("test.avi", "wb") { |f| f.write avi }
+
+    pipe_output("#{bin}/avimetaedit --out-tech test.avi")
```

Comment:
This actually tests nothing since `pipe_output` doesn't assert any particular exit code by default. You should probably use `shell_output` instead and also an `assert`.

Reply:
Fixed